packages/sdc/components/02-molecules/tooltip/tooltip.css

```
@@ -30,43 +40,60 @@
   width: calc(var(--ct-tooltip-triangle-height) * 2);
   height: var(--ct-tooltip-triangle-width);
 }
 .ct-tooltip .ct-tooltip__description[data-popper-placement^=top] > [data-tooltip-arrow]::before {
+  top: 0;
+  right: auto;
+  bottom: auto;
+  left: 0;
   border-left: var(--ct-tooltip-triangle-height) solid transparent;
   border-right: var(--ct-tooltip-triangle-height) solid transparent;
   border-top: var(--ct-tooltip-triangle-width) solid;
 }
```

Stylelint (17.14.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)